### DISCUSSION TITLE
Collect stats with ObservRTC

### DISCUSSION THREAD

**Discussion by misi** [edumeet/edumeet#823] [Ideas]
Deploy with edumeet an ObserRTC
* https://github.com/ObserveRTC/observer-js
* https://github.com/ObserveRTC/observer

**Comment by vpalmisano** [comment]
Another solution could be: https://github.com/fippo/rtcstats

Edit:
Tested right now and ObserRTC collected logs seem more complete.

**Comment by misi** [comment]
Few more links about architecture. There is no new release yet..
https://observertc.org/about/overview/
https://observertc.org/docs/

**Comment by misi** [comment]
browser and metrics implementations: https://webrtc-stats.callstats.io/ (reload the page multiple times if it does not appear for the first time)

**Comment by vpalmisano** [comment]
The GET request to `https://webrtc-stats.callstats.io/stats/scenario/list` fails after a timeout with 502 status.
